consentconstants/purposes.go

```diff
@@ -39,3 +40,87 @@ const (
 	// for you in other contexts, i.e. on other service, such as websites or apps, over time.
 	Measurement Purpose = 5
 )
+
+// TCF 2.0 Purposes:
+const (
```

Comment:
Should these go in /consentconstants/tcf2/ to match the vendor consent approach?